src/Raven.Server/Documents/Queries/LuceneQueryHelper.cs

```diff
@@ -137,7 +138,7 @@ public static Query Between(string fieldName, LuceneTermType termType, double fr
             return CreateRange(fieldName, fromValue, true, toValue, true);
         }
 
-        public static Query Term(string fieldName, string term, LuceneTermType type, float? boost = null, float? similarity = null, bool exact = false)
+        public static Query Term(string fieldName, string term, LuceneTermType type, float? boost = null, float? similarity = null, bool exact = false, bool lightWeight = false)
```

Comment:
I would rather make this into an instance, that has two static properties `Normal` and `Lightweight`

Reply:
So: `LuceneQueryHelper.Lightwegiht.Term(...)`